tests/general/LicenseCheckBearTest.py

```diff
@@ -0,0 +1,78 @@
+from queue import Queue
+import os.path
```

Comment:
We usually just do `import os` :)

Any particular reasons for `import os.path`?